src/main/java/org/apache/accumulo/testing/continuous/BulkIngest.java

```diff
@@ -103,157 +102,13 @@ public int run(String[] args) throws Exception {
         job.waitForCompletion(true);
         boolean success = job.isSuccessful();
 
-        // bulk import completed files
-        if (success) {
-          log.info("Sort and create job successful. Bulk importing {} to {}", RFILE_DIR, tableName);
-          client.tableOperations().importDirectory(RFILE_DIR).to(tableName).load();
-        } else {
-          log.error("Job failed, not calling bulk import");
-        }
         return success ? 0 : 1;
```

Comment:
Looks like you don't need `success` anymore

Reply:
I think its still useful for the process return code.